ui/components/component-library/text/text.test.js

```diff
@@ -110,19 +110,18 @@ describe('Text', () => {
   it('should render the Text with proper text color class name', () => {
     const { getByText } = render(
       <>
-        <Text color={COLORS.TEXT_DEFAULT}>text-default</Text>
-        <Text color={COLORS.TEXT_ALTERNATIVE}>text-alternative</Text>
-        <Text color={COLORS.TEXT_MUTED}>text-muted</Text>
-        <Text color={COLORS.OVERLAY_INVERSE}>overlay-inverse</Text>
```

Comment:
Any reason to remove this color and respective test?

Reply:
There was no TextColor under this key, i will add it back for transparency 🙏